<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>测试报告</title>
    <meta name="generator" content="HTMLTestRunner 0.9.1"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    
<style type="text/css" media="screen">
    body        { font-family: verdana, arial, helvetica, sans-serif; font-size: 80%; }
    table       { font-size: 100%; }
    pre         { white-space: pre-wrap;word-wrap: break-word; }

    /* -- heading ---------------------------------------------------------------------- */
    h1 {
        font-size: 16pt;
        color: gray;
    }
    .heading {
        margin-top: 0ex;
        margin-bottom: 1ex;
    }

    .heading .attribute {
        margin-top: 1ex;
        margin-bottom: 0;
    }

    .heading .description {
        margin-top: 2ex;
        margin-bottom: 3ex;
    }

    /* -- css div popup ------------------------------------------------------------------------ */
    a.popup_link {
    }

    a.popup_link:hover {
        color: red;
    }

    .popup_window {
        display: none;
        position: relative;
        left: 0px;
        top: 0px;
        /*border: solid #627173 1px; */
        padding: 10px;
        /*background-color: #E6E6D6; */
        font-family: "Lucida Console", "Courier New", Courier, monospace;
        text-align: left;
        font-size: 8pt;
        /* width: 500px;*/
    }

    }
    /* -- report ------------------------------------------------------------------------ */
    #show_detail_line {
        margin-top: 3ex;
        margin-bottom: 1ex;
    }
    #result_table {
        width: 99%;
    }
    #header_row {
        font-weight: bold;
        color: #303641;
        background-color: #ebebeb;
    }
    #total_row  { font-weight: bold; }
    .passClass  { background-color: #bdedbc; }
    .failClass  { background-color: #ffefa4; }
    .errorClass { background-color: #ffc9c9; }
    .passCase   { color: #6c6; }
    .failCase   { color: #FF6600; font-weight: bold; }
    .errorCase  { color: #c00; font-weight: bold; }
    .hiddenRow  { display: none; }
    .testcase   { margin-left: 2em; }


    /* -- ending ---------------------------------------------------------------------- */
    #ending {
    }

    #div_base {
                position:absolute;
                top:0%;
                left:5%;
                right:5%;
                width: auto;
                height: auto;
                margin: -15px 0 0 0;
    }
</style>

    <link href="css/bootstrap.min.css" rel="stylesheet">
    <script type="text/javascript" src="js/echarts.common.min.js"></script>
</head>
<body>
    <script language="javascript" type="text/javascript"><!--
    output_list = Array();

    /* level - 0:Summary; 1:Failed; 2:All */
    function showCase(level) {
        trs = document.getElementsByTagName("tr");
        for (var i = 0; i < trs.length; i++) {
            tr = trs[i];
            id = tr.id;
            if (id.substr(0,2) == 'ft') {
                if (level < 1) {
                    tr.className = 'hiddenRow';
                }
                else {
                    tr.className = '';
                }
            }
            if (id.substr(0,2) == 'pt') {
                if (level > 1) {
                    tr.className = '';
                }
                else {
                    tr.className = 'hiddenRow';
                }
            }
        }
    }


    function showClassDetail(cid, count) {
        var id_list = Array(count);
        var toHide = 1;
        for (var i = 0; i < count; i++) {
            tid0 = 't' + cid.substr(1) + '.' + (i+1);
            tid = 'f' + tid0;
            tr = document.getElementById(tid);
            if (!tr) {
                tid = 'p' + tid0;
                tr = document.getElementById(tid);
            }
            id_list[i] = tid;
            if (tr.className) {
                toHide = 0;
            }
        }
        for (var i = 0; i < count; i++) {
            tid = id_list[i];
            if (toHide) {
                document.getElementById('div_'+tid).style.display = 'none'
                document.getElementById(tid).className = 'hiddenRow';
            }
            else {
                document.getElementById(tid).className = '';
            }
        }
    }


    function showTestDetail(div_id){
        var details_div = document.getElementById(div_id)
        var displayState = details_div.style.display
        // alert(displayState)
        if (displayState != 'block' ) {
            displayState = 'block'
            details_div.style.display = 'block'
        }
        else {
            details_div.style.display = 'none'
        }
    }


    function html_escape(s) {
        s = s.replace(/&/g,'&amp;');
        s = s.replace(/</g,'&lt;');
        s = s.replace(/>/g,'&gt;');
        return s;
    }

    /* obsoleted by detail in <div>
    function showOutput(id, name) {
        var w = window.open("", //url
                        name,
                        "resizable,scrollbars,status,width=800,height=450");
        d = w.document;
        d.write("<pre>");
        d.write(html_escape(output_list[id]));
        d.write("\n");
        d.write("<a href='javascript:window.close()'>close</a>\n");
        d.write("</pre>\n");
        d.close();
    }
    */
    --></script>

    <div id="div_base">
        
    <div class='page-header'>
        <h1>测试报告</h1>
    <p class='attribute'><strong>开始时间:</strong> 2018-08-29 15:13:36</p>
<p class='attribute'><strong>运行时长:</strong> 0:02:41.412189</p>
<p class='attribute'><strong>状态:</strong> 错误 16</p>

    </div>
    <div style="float: left;width:50%;"><p class='description'>测试用例执行情况</p></div>
    <div id="chart" style="width:50%;height:400px;float:left;"></div>

        
    <div class="btn-group btn-group-sm">
        <button class="btn btn-default" onclick='javascript:showCase(0)'>总结</button>
        <button class="btn btn-default" onclick='javascript:showCase(1)'>失败</button>
        <button class="btn btn-default" onclick='javascript:showCase(2)'>全部</button>
    </div>
    <p></p>
    <table id='result_table' class="table table-bordered">
        <colgroup>
            <col align='left' />
            <col align='right' />
            <col align='right' />
            <col align='right' />
            <col align='right' />
            <col align='right' />
        </colgroup>
        <tr id='header_row'>
            <td>测试套件/测试用例</td>
            <td>总数</td>
            <td>通过</td>
            <td>失败</td>
            <td>错误</td>
            <td>查看</td>
        </tr>
        
    <tr class='errorClass'>
        <td>unittest.loader._FailedTest</td>
        <td>1</td>
        <td>0</td>
        <td>0</td>
        <td>1</td>
        <td><a href="javascript:showClassDetail('c1',1)">详情</a></td>
    </tr>

<tr id='ft1.1' class='none'>
    <td class='errorCase'><div class='testcase'>Test_login</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.1')" >
        错误</a>

    <div id='div_ft1.1' class="popup_window">
        <pre>
        ft1.1: ImportError: Failed to import test module: Test_login
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "D:\work\AutoTestofVantpy\TestCase\Test_login.py", line 19
    @classmethod
               ^
IndentationError: unindent does not match any outer indentation level


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

    <tr class='errorClass'>
        <td>Test_movelabel.moveuserCase: 我的账号下拉标签跳转</td>
        <td>7</td>
        <td>0</td>
        <td>0</td>
        <td>7</td>
        <td><a href="javascript:showClassDetail('c2',7)">详情</a></td>
    </tr>

<tr id='ft2.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_movebrower: 跳转至我的借款</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.1')" >
        错误</a>

    <div id='div_ft2.1' class="popup_window">
        <pre>
        ft2.1: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_movelabel.py", line 94, in test_movebrower
    if title(self.driver) == True:
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 38, in __call__
    return self.title == driver.title
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 341, in title
    resp = self.execute(Command.GET_TITLE)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_movekaquan: 跳转至我的卡券</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.2')" >
        错误</a>

    <div id='div_ft2.2' class="popup_window">
        <pre>
        ft2.2: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_movelabel.py", line 182, in test_movekaquan
    FP.click_top_login()
  File "..\Pages\FirstPage.py", line 75, in click_top_login
    self.click(self.top_login)
  File "..\Base\Selenium2.py", line 74, in click
    self.find_element(*loc).click()
  File "..\Base\Selenium2.py", line 34, in find_element
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "..\Base\Selenium2.py", line 34, in &lt;lambda&gt;
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 966, in find_element
    'value': value})['value']
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.3' class='none'>
    <td class='errorCase'><div class='testcase'>test_movepay: 跳转至我的出借</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.3')" >
        错误</a>

    <div id='div_ft2.3' class="popup_window">
        <pre>
        ft2.3: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_movelabel.py", line 63, in test_movepay
    if title(self.driver) == True:
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 38, in __call__
    return self.title == driver.title
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 341, in title
    resp = self.execute(Command.GET_TITLE)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.4' class='none'>
    <td class='errorCase'><div class='testcase'>test_movepaylog: 跳转至交易记录</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.4')" >
        错误</a>

    <div id='div_ft2.4' class="popup_window">
        <pre>
        ft2.4: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_movelabel.py", line 124, in test_movepaylog
    if title(self.driver) == True:
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 38, in __call__
    return self.title == driver.title
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 341, in title
    resp = self.execute(Command.GET_TITLE)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.5' class='none'>
    <td class='errorCase'><div class='testcase'>test_movesetting: 跳转至账号设置</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.5')" >
        错误</a>

    <div id='div_ft2.5' class="popup_window">
        <pre>
        ft2.5: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_movelabel.py", line 154, in test_movesetting
    if title(self.driver) == True:
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 38, in __call__
    return self.title == driver.title
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 341, in title
    resp = self.execute(Command.GET_TITLE)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.6' class='none'>
    <td class='errorCase'><div class='testcase'>test_moveuser: 跳转至账号概览</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.6')" >
        错误</a>

    <div id='div_ft2.6' class="popup_window">
        <pre>
        ft2.6: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_movelabel.py", line 33, in test_moveuser
    if title(self.driver) == True:
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 38, in __call__
    return self.title == driver.title
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 341, in title
    resp = self.execute(Command.GET_TITLE)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.7' class='none'>
    <td class='errorCase'><div class='testcase'>test_moveyaoqing: 跳转至我的邀请</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.7')" >
        错误</a>

    <div id='div_ft2.7' class="popup_window">
        <pre>
        ft2.7: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_movelabel.py", line 216, in test_moveyaoqing
    FP.move_label_myuser()#悬浮我的账号
  File "..\Pages\FirstPage.py", line 108, in move_label_myuser
    self.move_to_element(self.label_myuser)		#悬浮
  File "..\Base\Selenium2.py", line 97, in move_to_element
    element = self.find_element(*loc)
  File "..\Base\Selenium2.py", line 34, in find_element
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "..\Base\Selenium2.py", line 34, in &lt;lambda&gt;
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 966, in find_element
    'value': value})['value']
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

    <tr class='errorClass'>
        <td>unittest.suite._ErrorHolder</td>
        <td>3</td>
        <td>0</td>
        <td>0</td>
        <td>3</td>
        <td><a href="javascript:showClassDetail('c3',3)">详情</a></td>
    </tr>

<tr id='ft3.1' class='none'>
    <td class='errorCase'><div class='testcase'>newallCase)</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.1')" >
        错误</a>

    <div id='div_ft3.1' class="popup_window">
        <pre>
        ft3.1: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_newall.py", line 18, in setUpClass
    cls.driver = driver.openbrowser(cls)
  File "..\Base\BrowserDriver.py", line 43, in openbrowser
    driver.get(url)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 332, in get
    self.execute(Command.GET, {'url': url})
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: cannot determine loading status
from unknown error: cannot determine loading status
from disconnected: received Inspector.detached event
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft3.2' class='none'>
    <td class='errorCase'><div class='testcase'>removeabCase)</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.2')" >
        错误</a>

    <div id='div_ft3.2' class="popup_window">
        <pre>
        ft3.2: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_removeab.py", line 19, in setUpClass
    cls.driver = driver.openbrowser(cls)
  File "..\Base\BrowserDriver.py", line 36, in openbrowser
    driver = webdriver.Chrome(self.chrome_driver_path)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 75, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 156, in __init__
    self.start_session(capabilities, browser_profile)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 251, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: session not created exception
from disconnected: received Inspector.detached event
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft3.3' class='none'>
    <td class='errorCase'><div class='testcase'>removehpCase)</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.3')" >
        错误</a>

    <div id='div_ft3.3' class="popup_window">
        <pre>
        ft3.3: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_removehelp.py", line 18, in setUpClass
    cls.driver = driver.openbrowser(cls)
  File "..\Base\BrowserDriver.py", line 36, in openbrowser
    driver = webdriver.Chrome(self.chrome_driver_path)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 75, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 156, in __init__
    self.start_session(capabilities, browser_profile)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 251, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: session not created exception
from chrome not reachable
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

    <tr class='errorClass'>
        <td>Test_removelogin.removelgCase</td>
        <td>1</td>
        <td>0</td>
        <td>0</td>
        <td>1</td>
        <td><a href="javascript:showClassDetail('c4',1)">详情</a></td>
    </tr>

<tr id='ft4.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_removelogin</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.1')" >
        错误</a>

    <div id='div_ft4.1' class="popup_window">
        <pre>
        ft4.1: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_removelogin.py", line 26, in test_removelogin
    FP.click_top_login()	#跳转至登录界面
  File "..\Pages\FirstPage.py", line 75, in click_top_login
    self.click(self.top_login)
  File "..\Base\Selenium2.py", line 74, in click
    self.find_element(*loc).click()
  File "..\Base\Selenium2.py", line 34, in find_element
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "..\Base\Selenium2.py", line 34, in &lt;lambda&gt;
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 966, in find_element
    'value': value})['value']
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

    <tr class='errorClass'>
        <td>Test_removezc.removezcCase</td>
        <td>1</td>
        <td>0</td>
        <td>0</td>
        <td>1</td>
        <td><a href="javascript:showClassDetail('c5',1)">详情</a></td>
    </tr>

<tr id='ft5.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_removezc</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.1')" >
        错误</a>

    <div id='div_ft5.1' class="popup_window">
        <pre>
        ft5.1: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_removezc.py", line 25, in test_removezc
    FP.click_top_zhuce() #跳转至注册界面
  File "..\Pages\FirstPage.py", line 78, in click_top_zhuce
    self.click(self.top_zhuce)
  File "..\Base\Selenium2.py", line 74, in click
    self.find_element(*loc).click()
  File "..\Base\Selenium2.py", line 34, in find_element
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

    <tr class='errorClass'>
        <td>Test_swtichlabel.swtichlbCase</td>
        <td>1</td>
        <td>0</td>
        <td>0</td>
        <td>1</td>
        <td><a href="javascript:showClassDetail('c6',1)">详情</a></td>
    </tr>

<tr id='ft6.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_swtichlb</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.1')" >
        错误</a>

    <div id='div_ft6.1' class="popup_window">
        <pre>
        ft6.1: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_swtichlabel.py", line 26, in test_swtichlb
    FP.click_label_dq()#切换定期标签
  File "..\Pages\FirstPage.py", line 96, in click_label_dq
    self.click(self.label_dq)
  File "..\Base\Selenium2.py", line 74, in click
    self.find_element(*loc).click()
  File "..\Base\Selenium2.py", line 34, in find_element
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "..\Base\Selenium2.py", line 34, in &lt;lambda&gt;
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
AttributeError: 'NoneType' object has no attribute 'is_displayed'

        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

    <tr class='errorClass'>
        <td>Test_swtichqkpay.removelgCase</td>
        <td>1</td>
        <td>0</td>
        <td>0</td>
        <td>1</td>
        <td><a href="javascript:showClassDetail('c7',1)">详情</a></td>
    </tr>

<tr id='ft7.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_swtichqkpay</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.1')" >
        错误</a>

    <div id='div_ft7.1' class="popup_window">
        <pre>
        ft7.1: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_swtichqkpay.py", line 26, in test_swtichqkpay
    FP.scroll_middle()
  File "..\Pages\FirstPage.py", line 54, in scroll_middle
    self.js_scroll_middle()
  File "..\Base\Selenium2.py", line 179, in js_scroll_middle
    self.driver.execute_script(js)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 635, in execute_script
    'args': converted_args})['value']
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

    <tr class='errorClass'>
        <td>Test_urljudge.removelgCase</td>
        <td>1</td>
        <td>0</td>
        <td>0</td>
        <td>1</td>
        <td><a href="javascript:showClassDetail('c8',1)">详情</a></td>
    </tr>

<tr id='ft8.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_judge_url</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft8.1')" >
        错误</a>

    <div id='div_ft8.1' class="popup_window">
        <pre>
        ft8.1: Traceback (most recent call last):
  File "D:\work\AutoTestofVantpy\TestCase\Test_urljudge.py", line 28, in test_judge_url
    FP.click_help_jfpt()
  File "..\Pages\FirstPage.py", line 162, in click_help_jfpt
    self.click(self.help_jfpt)
  File "..\Base\Selenium2.py", line 74, in click
    self.find_element(*loc).click()
  File "..\Base\Selenium2.py", line 34, in find_element
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "..\Base\Selenium2.py", line 34, in &lt;lambda&gt;
    WebDriverWait(self.driver, 30).until(lambda driver: driver.find_element(*loc).is_displayed())
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 966, in find_element
    'value': value})['value']
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=69.0.3493.3)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.17134 x86_64)


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

        <tr id='total_row'>
            <td>总计</td>
            <td>16</td>
            <td>0</td>
            <td>0</td>
            <td>16</td>
            <td>&nbsp;</td>
        </tr>
    </table>

        <div id='ending'>&nbsp;</div>
        
    <script type="text/javascript">
        // 基于准备好的dom，初始化echarts实例
        var myChart = echarts.init(document.getElementById('chart'));

        // 指定图表的配置项和数据
        var option = {
            title : {
                text: '测试执行情况',
                x:'center'
            },
            tooltip : {
                trigger: 'item',
                formatter: "{a} <br/>{b} : {c} ({d}%)"
            },
            legend: {
                orient: 'vertical',
                left: 'left',
                data: ['通过','失败','错误']
            },
            series : [
                {
                    name: '测试执行情况',
                    type: 'pie',
                    radius : '60%',
                    center: ['50%', '60%'],
                    data:[
                        {value:0, name:'通过'},
                        {value:0, name:'失败'},
                        {value:16, name:'错误'}
                    ],
                    itemStyle: {
                        emphasis: {
                            shadowBlur: 10,
                            shadowOffsetX: 0,
                            shadowColor: 'rgba(0, 0, 0, 0.5)'
                        }
                    }
                }
            ]
        };

        // 使用刚指定的配置项和数据显示图表。
        myChart.setOption(option);
    </script>
    
    </div>
</body>
</html>
